python/cudf/cudf/_lib/copying.pyx

```diff
@@ -204,7 +204,7 @@ def _scatter_scalar(scalars, Column scatter_map,
     cdef Scalar slr
     for val, col in zip(scalars, target_table._columns):
         slr = as_scalar(val, col.dtype)
-        source_scalars.push_back(move(slr.c_value))
+        source_scalars.push_back(move(slr.get_uptr()._device_uptr))
```

Comment:
Can we raise an issue that the libcudf API here should probably be updated to take in a `vector<scalar*>` instead of `vector<unique_ptr<scalar>>`?

Reply:
Sure - can do. This is the only such place where this is needed, so it probably makes sense to update it on the libcudf side.

Reply:
created https://github.com/rapidsai/cudf/issues/6554

Reply:
Can we add a comment with a link to that issue in the code to make it easy to reference / find?